core/src/main/java/org/radargun/stages/monitor/AbstractMonitorStartStage.java

```diff
@@ -0,0 +1,85 @@
+package org.radargun.stages.monitor;
+
+import org.radargun.DistStageAck;
+import org.radargun.config.Property;
+import org.radargun.reporting.Timeline;
+import org.radargun.stages.AbstractDistStage;
+import org.radargun.sysmonitor.AbstractMonitors;
+import org.radargun.sysmonitor.CpuUsageMonitor;
+import org.radargun.sysmonitor.GcMonitor;
+import org.radargun.sysmonitor.InternalsMonitor;
+import org.radargun.sysmonitor.MemoryUsageMonitor;
+import org.radargun.sysmonitor.NetworkBytesMonitor;
+import org.radargun.sysmonitor.OpenFilesMonitor;
+import org.radargun.sysmonitor.RssMonitor;
+import org.radargun.sysmonitor.SystemWorkerMonitor;
+import org.radargun.sysmonitor.ThreadDumpMonitor;
+import org.radargun.traits.InternalsExposition;
+import org.radargun.traits.JmxConnectionProvider;
+import org.radargun.utils.SystemUtils;
+import org.radargun.utils.TimeConverter;
+
+/**
+ *
+ * Starts collecting JVM statistics locally on main and each worker node.
+ * {@link SystemWorkerMonitor}
+ *
+ * @author Alan Field &lt;afield@redhat.com&gt;
```

Comment:
Not sure I should be the `@author` here!

Reply:
The `AbstractMonitorStartStage` is basically `MonitorStartStage` that has your name. I decided to keep it :)